pkg/operator/ceph/nfs/config.go

```diff
@@ -50,7 +50,7 @@ func getNFSNodeID(n *cephv1.CephNFS, name string) string {
 
 func getGaneshaConfigObject(n *cephv1.CephNFS, version cephver.CephVersion, name string) string {
 	if version.IsAtLeastOctopus() {
```

Comment:
@travisn Do you want a comment here too ?

Reply:
Yes, a comment here would be helpful for future reference, thanks

Reply:
Done!